### ISSUE TITLE
Improve color contrast

### ISSUE THREAD

**Issue by mgifford** [formbricks/formbricks#1550]
### Issue Summary

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (#role-skip)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Formbricks - https://app.formbricks.com/onboarding

Element path: #role-skip

Snippet: <button class="inline-flex items-center appearance-none px-4 py-2 text-base font-medium rounded-md relative hover:text-slate-600 focus:outline-none focus:ring-2 focus:ring-offset-1 focus:ring-neutral-900 dark:text-slate-700 dark:hover:text-slate-500 text-slate-400" id="role-skip">Skip</button>

Related paths: 
  body > .h-full.bg-slate-50.w-full

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 2.45 (foreground color: #94a3b8, background color: #f8fafc, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 118.0.2088.76

====

This accessibility issue was found using Accessibility Insights for Web 2.40.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Steps to Reproduce

I just loaded the https://app.formbricks.com/onboarding page

### Expected behavior

There should be clear color contrast especially for links.

And links should be underlined. 

### Screenshots

![Screenshot 2023-11-01 at 6 59 23 PM](https://github.com/formbricks/formbricks/assets/116832/8ca26227-4c46-4430-a642-2c56474f9cd4)

### Environment

- [X] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS:
- Node:
- npm:
```

**Comment by AndydeCleyre** [comment]
There are still areas where the foreground color is not set at all, and uses the (unpredictable) individuals' browser fallback foreground preference.

If the user has set fallbacks for unspecified colors to light-on-dark, the fact that formbricks doesn't specify the foreground but does specify the background results in light-on-light.

![image](https://github.com/user-attachments/assets/dcdaedaa-fe96-44fe-902f-f16793238eb1)

![image](https://github.com/user-attachments/assets/2cfdbdf1-9621-4e70-b52b-e4c972d60fc9)

![image](https://github.com/user-attachments/assets/8743b9f1-b4a2-4779-9896-7f989d326591)

![image](https://github.com/user-attachments/assets/1186a004-e113-4d40-8f4b-a36f6813e8e9)

![image](https://github.com/user-attachments/assets/410caeeb-6c89-4c0b-b774-7686f663540e)
